src/openrct2/interface/Viewport.cpp

```diff
@@ -655,11 +655,11 @@ void viewport_update_smart_sprite_follow(rct_window * window)
 
 void viewport_update_smart_guest_follow(rct_window * window, rct_peep * peep)
 {
-    union
+    struct
     {
         sprite_focus sprite;
         coordinate_focus coordinate;
```

Comment:
If this is not a `union`, I don't see a point for it to be a `struct`. In fact, `focus.coordinate` is only ever written to and if you split the fields to their own variables, I'm pretty sure compilers won't let it pass.

Reply:
Done. Question, read comment.